pyscf/neo/tdgrad.py

```diff
@@ -464,11 +464,11 @@ def grad_elec_uhf(td_grad, x_y, atmlst=None, max_memory=2000, verbose=logger.INF
                                             mo_coeff['e'], mo_occ['e'], spin=1)
         f1vo, f1oo, vxc1, k1ao = \
                 tduks._contract_xc_kernel(td_grad_e, mf_e.xc, (dmxpya,dmxpyb),
-                                    (dmzooa,dmzoob), True, True, max_memory)
+                                         (dmzooa,dmzoob), True, True, max_memory)
```

Comment:
Need to add one more space. Align `(dmzooa,dmzoob)` with `td_grad_e`, not `(` with `(`.